build-extras.gradle

```diff
@@ -0,0 +1,7 @@
+android.applicationVariants.all { variant ->
+    variant.outputs.all {
+        outputFileName = outputFileName
+            .replace("app-", "jellyfin-android_")
```

Comment:
not a fan of mixing dash and underscore here

Reply:
Well that's the convention that has been used since the first beta... https://repo.jellyfin.org/releases/client/android/versions/beta1/ :joy:

Reply:
and I propose we change that at some point :p definitely not a blocker though